geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapred/InputFormatBaseAdapter.scala

```diff
@@ -1,15 +1,14 @@
 package org.locationtech.geomesa.jobs.mapred
 
 import org.apache.accumulo.core.client.mapred.AccumuloInputFormat
-import org.apache.accumulo.core.client.security.tokens.PasswordToken
+import org.apache.accumulo.core.client.security.tokens.{AuthenticationToken, PasswordToken}
```

Comment:
Can we add a unit test for this class?  May not be possible especially since it requires a matrix of deps to run against.  I don't really know how to do that.